packages/genesys-spark/snapshot/scss/_mixins.scss

```diff
@@ -28,8 +28,8 @@
 @mixin gux-sr-only-clip {
   &:not(:focus):not(:active) {
     position: absolute;
-    width: 1px;
-    height: 1px;
+    width: 0;
+    height: 0;
```

Comment:
I don't think we can do this. I think the element has to have some height and width to be rendered

Reply:
@daragh-king-genesys It's the screenreader text, we don't want it rendered, or am I missing something?
Or is there a difference between rendered & visible?

Reply:
Ya have a read of this https://webaim.org/techniques/css/invisiblecontent/

Reply:
@daragh-king-genesys Thanks. I think I don't need it anyway.